### PR TITLE
fix(api): case where description was not returned

### LINKED ISSUE #1514: Résultats sans description dans la recherche
Certaines fiches SP n'ont pas de description dans la recherche. C'est problématique car cela permet de différencier des fiches avec des titres identiques ou proches.

image

### PR DESCRIPTION
**Pull request by UnbearableBear**
Fix #1514 

I wonder if we should add some test for this not to happen anymore. Or maybe collocate the `_source` attribute ?

### CODE REVIEW

**Comment by lionelB** [comment]
I would add a test case (using a preset vector for the query and a presetVector for a document) and add a test case for getSemSearch 
- to retrieve the vector for a document, extract a document with a vector from a dump file from nlp or data container and
- to retrive a vector for a query use http://nlp.master.code-du-travail-numerique.dev.factory.social.gouv.fr/api/search?q=d%C3%A9mission

**Review by lionelB — APPROVED** [review]

**Comment by UnbearableBear** [comment]
J'ai rien compris à ce que tu proposes mais je merge !

**Review by nkrmr — APPROVED** [review]

File: packages/code-du-travail-api/src/server/routes/__tests__/search.test.js [hunk 1]
```diff
@@ -11,7 +14,11 @@ const fetch = require("node-fetch");
 const app = new Koa();
 app.use(router.routes());
 
-test("return search results for demission from datafiller", async () => {
+it("asks same sources wether it is search sem or search elastic and gets a description", () => {
```
Comment (UnbearableBear) [inline packages/code-du-travail-api/src/server/routes/__tests__/search.test.js]: @lionelB This should do the trick ?

File: packages/code-du-travail-api/src/server/routes/__tests__/search.test.js [hunk 2]
```diff
@@ -15,7 +15,10 @@ const app = new Koa();
 app.use(router.routes());
 
 it("asks same sources wether it is search sem or search elastic and gets a description", () => {
```
Comment (UnbearableBear) [inline packages/code-du-travail-api/src/server/routes/__tests__/search.test.js:17]: @lionelB This should do the trick ?

Comment (lionelB) [reply packages/code-du-travail-api/src/server/routes/__tests__/search.test.js:17]: nice trick !  We will probably need to update mock data but for now, job is done !

**Review by lionelB — APPROVED** [review]

**Pull request merged**